src/index.html

```diff
@@ -9,10 +9,53 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="/src/style.css"
     />
```

Comment:
Keep your attributes correctly formatted. Since the `link` element has more than two attributes, start each one on a new line with 2-space indentation related to the tag.